dev-python/pytest-sanic/pytest-sanic-1.6.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_6 )
+
+inherit distutils-r1
+
+DESCRIPTION="a Pytest Plugin for Sanic"
+HOMEPAGE="
+	https://pypi.python.org/pypi/pytest-sanic
+	https://github.com/yunstanford/pytest-sanic
+"
+SRC_URI="https://github.com/yunstanford/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
```

Comment:
`IUSE="test"` and `RESTRICT=..` is already added by calling the `distutils_enable_tests` function
`IUSE="doc"` is added byt the `distutils_enable_sphinx` function

so these lines can go